ddcloud/resource_server.go

```diff
@@ -593,6 +655,12 @@ func resourceServerImport(data *schema.ResourceData, provider interface{}) (impo
 		models.NewDisksFromVirtualMachineSCSIControllers(server.SCSIControllers),
 	)
 
+	if server.Started {
+		data.Set(resourceKeyServerPowerState, "start")
+	} else {
+		data.Set(resourceKeyServerPowerState, "Stop")
```

Comment:
Is `Stop` meant to be `stop`?

Reply:
Oh, just realised: if this is run during import, then we might want to check to see if they've specified power state in the config at all before adding this value to the state (otherwise someone who hasn't specified power state will wind up "opting in" to power state management without realising it when they import a `ddcloud_server`).

It's fine for now, but we might want to revisit this once these changes are merged (or I can if you have better things to do :wink:).

Reply:
@tintoy good catch. The import code is something i missed in my clean up after changing the power_state options. I'm changing that to mean resourceKeyServerStarted  = true / false.

I think it is best not to set the power_state when importing. That way it'll either be default (disabled) or whatever they configure it to be. That way it won't trip anyone up.